plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -271,7 +271,16 @@ object ScalafmtPlugin extends AutoPlugin {
     val rootBase = (LocalRootProject / baseDirectory).value
     val thisBase = (thisProject.value).base
     if (rootBase == thisBase)
-      (BuildPaths.projectStandard(thisBase) ** GlobFilter("*.scala")).get
+      BuildPaths
+        .projectStandard(thisBase)
+        .descendantsExcept(
+          "*.scala",
+          (pathname: File) =>
+            pathname.getAbsolutePath
+              .drop(thisBase.getAbsolutePath.length)
+              .contains("/target/")
```

Comment:
Maybe we can use [outputDirectory(projectStandard(thisBase))](https://github.com/sbt/sbt/blob/a8e319e97b3c800915382e2c9528edf7a54efb65/main/src/main/scala/sbt/BuildPaths.scala#L121) for filtering them.

Reply:
Thank you so much, I didn't know about it!